pkg/eventcache/eventcache.go

```diff
@@ -59,10 +64,15 @@ func handleExecEvent(event *cacheObj, nspid uint32) error {
 		podInfo, _ = process.GetPodInfo(containerId, filename, args, nspid)
 		if podInfo == nil {
 			errormetrics.ErrorTotalInc(errormetrics.EventCachePodInfoRetryFailed)
-			return fmt.Errorf("failed to get pod info")
+			return ErrFailedToGetPodInfo
 		}
 	}
 
+	if event.internal == nil {
```

Comment:
This would just be Exit events correct? Execve events should have this?

Reply:
@jrfastab This handler is only being run on exec events right now. Should it be run for exit events too?

Reply:
no its good as is.... :coffee:

Reply:
shouldn't we call process.GetParentProcessInternal(p.Pid.Value, event.timestamp) here? Where are we going to get out of this loop?